MediaBrowser.Api/Playback/Hls/HLSCodecStringFactory.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Text;
+using System.Linq;
+
+
+namespace MediaBrowser.Api.Playback
+{
+    /// <summary>
+    /// Get various codec strings for use in HLS playlists
+    /// </summary>
+    static class HLSCodecStringFactory
+    {
+        private enum MaxVideoStringLength
```

Comment:
Why are these enums?

Reply:
They're not really needed anymore so I went ahead and removed them. For whenever I get the bad idea to abuse enums like this again though, what would be a better solution, a const Dictionary<string, int>?

Reply:
Just some `const`s

Reply:
👍